combat/CombatSystem.cpp

```diff
@@ -273,22 +275,44 @@ namespace {
             part_type_name(part_name_),
             part_attack(part_attack_),
             fighters_launched(0),
-            fighter_damage(0.0f)
+            fighter_damage(0.0f),
```

Comment:
prefer moving default values such as this (with any preexisting) into the declarations rather than extending the initialization list.

Reply:
That constructor wasn't necessary actually. So i removed it.